src/modules/design/bpmn-io/bpmn-io.ts

```diff
@@ -405,6 +414,30 @@ export class BpmnIo {
     return returnPromise;
   }
 
+  private _renameFormFields(event: IInternalEvent): IInternalEvent {
+    const formFields: Array<IModdleElement> = event.descriptor.businessObject.extensionElements.values[0].fields.filter((field: IModdleElement) => {
```

Comment:
I would split this up a bit:
```suggestion
    extensionElements: YourTypeHere = event.descriptor.buissnesObject.extensionElements.values
    
    if (extensionElements.length() === 0) {
       //Handle Error here
    }
    
    const formFieldExtensionElements: YourTypeHere = extensionElements[0];
    const formFields: Array<IModdleElement> = formFieldExtensionElements.fields.filter((field: IModdleElement) => {
```

Reply:
I would just do it like this:
```suggestion
    const allFormFields: Array<IModdleElement> = event.descriptor.businessObject.extensionElements.values[0].fields;

    const filteredFormFields: Array<IModdleElement> = allFormFields.filter((field: IModdleElement) => {
```

Reply:
Can we ensure that the retrieved array always has at least one entry? If so then do it like @SteffenKn staid.

Reply:
@roblen45 Yes

Reply:
@robpal94 Yes